components/style/attr.rs

```diff
@@ -5,12 +5,13 @@
 use app_units::Au;
 use cssparser::{self, Color, RGBA};
 use euclid::num::Zero;
+use num::ToPrimitive;
 use std::ascii::AsciiExt;
 use std::ops::Deref;
 use string_cache::{Atom, Namespace};
 use url::Url;
-use util::str::{DOMString, LengthOrPercentageOrAuto, WHITESPACE, parse_unsigned_integer, parse_length};
-use util::str::{split_html_space_chars, str_join, parse_integer};
+use util::str::{DOMString, HTML_SPACE_CHARACTERS, WHITESPACE, LengthOrPercentageOrAuto};
```

Comment:
nit: Put the constants to the right, and bring `LengthOrPercentageOrAuto` right after `DOMString`.